ci/pkl-pipelines/pay-deploy/node-runner.pkl

```diff
@@ -58,7 +58,7 @@ jobs {
                 }
                 params {
                     ["CONTEXT"] = "node-runner-src/ci/docker/node-runner"
-                    ["DOCKERFILE"] = "Dockerfile.node18"
+                    ["DOCKERFILE"] = "node-runner-src/ci/docker/node-runner/Dockerfile.node18"
```

Comment:
Something weird going on here....... the CONTEXT is supposed to be the directory where the commands are run 🤔

Reply:
Yeeeeah, I thought it was weird too. Might be some implementation weirdness of the oci-builder/buildkit.